.github/workflows/biotools-gh2biotools.yml

```diff
@@ -0,0 +1,123 @@
+name: Validate & run gh2biotools on .biotools.json changes
+
+on:
+  push:
+    paths:
+      - 'data/**/*.biotools.json'
+    branches:
+      - master
+
+jobs:
+  run-gh2biotools:
+    runs-on: ubuntu-latest
+    if: github.actor != 'biotools-bot'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          sparse-checkout: 'data'
+          sparse-checkout-cone-mode: false
+          fetch-depth: 0
+
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+
+      - name: Check for updated/created .biotools.json files
+        id: check_changes
+        run: |
+          files=$(git diff-tree --no-commit-id --name-only --diff-filter=d -r ${{ github.sha }} | grep '\.biotools\.json$' || true)
+
+          if [ -z "$files" ]; then
+            echo "No relevant biotools.json files changed."
+            echo "changed=false" >> $GITHUB_OUTPUT
+            exit 0
+          else
+            echo "Files changed:"
+            echo "$files"
+            echo "changed=true" >> $GITHUB_OUTPUT
+            echo "files=${files}" >> $GITHUB_OUTPUT
+          fi
+
+
+      - name: Validate allowed fields in modified biotools.json files
+        if: steps.check_changes.outputs.changed == 'true'
+        run: |
+          echo "Validating fields in changed .biotools.json files..."
+          PROTECTED_FIELDS=(
+            "biotoolsID"
+            "biotoolsCURIE"
+            "additionDate"
+            "collectionID"
+            "elixirPlatform"
+            "elixirNode"
+            "elixirCommunity"
+            "lastUpdate"
```

Comment:
will "last update" be updated on bio.tools automatically when posting the file to the endpoint?